crossbeam-channel/src/channel.rs

```diff
@@ -665,6 +665,81 @@ impl<T> Receiver<T> {
         }
     }
 
+    /// Attempts to receive multiple messages from the channel without blocking.
+    ///
+    /// This method will either receive multiple messages from the channel immediately, up to `limit`,
+    /// or return an error if the channel is empty.
```

Comment:
This can also return a `Disconnected` error, so maybe change the wording of this a little.

Reply:
Done.